opengrok-indexer/src/main/java/org/opengrok/indexer/index/Indexer.java

```diff
@@ -278,6 +276,8 @@ public static int runMain(String[] argv) {
                 }
             }
 
+            checkConfiguration(cfg);
```

Comment:
Moved to just before the call to `setConfiguration()` so that it could reflect the `cfg` changes done above.